graphgrid-sdk-java-tests/src/test/java/com/graphgrid/sdk/TestBase.java

```diff
@@ -8,14 +10,79 @@
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import java.io.IOException;
+import java.util.Properties;
+
 @ActiveProfiles( "test" )
 @RunWith( SpringRunner.class )
 @SpringBootTest( classes = App.class, properties = "server.port:0" )
 @Rollback( false )
 public abstract class TestBase
 {
 
+    // only used to test getting a token for a user
```

Comment:
this should possible go into docs to show how to configure withs spring/ param store